packages/resolvers/glob/src/GlobResolver.js

```diff
@@ -94,22 +94,14 @@ export default (new Resolver({
         logger,
       });
 
-      let ctx = {
-        invalidateOnFileCreate,
-        invalidateOnFileChange,
-        specifierType: dependency.specifierType,
-        loc: dependency.loc,
-        range: dependency.range,
-      };
-
       let result;
       try {
-        result = await resolver.resolveModule({
-          filename: pkg,
+        result = await resolver.resolve({
+          filename: pkg + '/package.json',
```

Comment:
Not sure how we should handle packages with an exports field here. This won't work in that case, and anyway we wouldn't want the glob to match based on files, it would need to be based on what is exposed by the exports field. Not sure if that would work in all cases.

Reply:
The case of `import("@org/some-package/*")` is probably also rather uncommon? So I think it's fine to keep it using files for now.